Library/Formula/bento4.rb

```diff
@@ -0,0 +1,31 @@
+class Bento4 < Formula
+  desc "Full-featured MP4 format and MPEG DASH library and tools"
+  homepage "https://www.bento4.com"
+  url "https://codeload.github.com/axiomatic-systems/Bento4/tar.gz/v1.4.3-607"
+  version "1.4.3-607"
+  sha256 "37d98fb105d5800e5e08f3cb304f8baeee020ea3599bf20cab2ff340737e5f45"
+  head "https://github.com/axiomatic-systems/Bento4.git"
+
+  depends_on :xcode => :build
+
+  conflicts_with "gpac", :because => "gpac also ships a MP42aac binary"
```

Comment:
I'd actually thought of doing this in a git-like manner, with binaries being prefixed with `bento4-`, and having a single `bento4` binary which looks for the relevant prefixed binary.